### DISCUSSION TITLE
How to contribute? Not able to access the website files

### DISCUSSION THREAD

**Discussion by vansh-codes** [recodehive/awesome-github-profiles#355] [Q&A] [answered]
I am not getting public or src folder or the website files like index.html .css .js files
I wanted to contribute please help me out how can I fix this issue

### Heres the file I have access to ⬇️ 

![image](https://github.com/user-attachments/assets/16bb1695-e50f-4a3d-9f9d-1a588268aa5d)

**Comment by sanjay-kv** [comment, marked as answer]
check gh-page branch

**Reply by vansh-codes** [reply]
Ohh wow got it! Thank you @sanjay-kv 🙌

**Comment by Ali-Hassan2** [comment]
@sanjay-kv Brother, I wanted to ask how I can unlock the Galaxy Brain achievement on GitHub. I answered two questions in Discussions but they were not marked as the answer is there something specific I need to do or follow?

**Reply by Whitfrost21** [reply]
make sure that you answered in active discussions , they were not marked means there are no active maintainers of that repo or the owner of that discussion is also not active yet. also replying to a comment don't mean you've have answered the Question. try answering in active repo's
